recipe/meta.yaml

```diff
@@ -20,6 +20,8 @@ requirements:
     - setuptools ==68.0.0
     - tomlkit ==0.11.8
     - pip
+    # see if wheel fixes build issues
+    - wheel
```

Comment:
@pvanheus and @thebestnom, would you be able to look into why `wheel` is required and whether than and be removed as a build-time requirement in a future release?